pkg/service/store/store.go

```diff
@@ -89,6 +91,11 @@ func (s *ClusterService) NamespaceServiceName() string {
 	return s.Namespace + "/" + s.Name
 }
 
+// ServiceNamespacedName returns the namespace and service name with types.NamespacedName
+func (s *ClusterService) ServiceNamespacedName() types.NamespacedName {
+	return types.NamespacedName{Name: s.Name, Namespace: s.Namespace}
+}
```

Comment:
Sorry, I just realized that I never published this comment on the other PR:

> I'd personally avoid to have both NamespaceServiceName() and ServiceNamespacedName(), as possibly confusing. What about modifying NamespaceServiceName() to return a NamespacedName, and convert possible remaining calling sites (there shouldn't be any other except from the ones you already modified) over to NamespaceServiceName().String() if a string is needed?

Reply:
Ah yesss, I checked and it was the only place used it seems... I supposed it was used in many places but not even :sweat_smile:.